src/index.ts

```diff
@@ -0,0 +1,97 @@
+import express, { Request, Response } from 'express';
+import swaggerJsdoc, { OAS3Definition } from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+
+import { Application } from 'express';
```

Comment:
There is already import from express, so you can merge these lines.

```suggestion
import express, { Application, Request, Response } from 'express';
import swaggerJsdoc, { OAS3Definition } from 'swagger-jsdoc';
import swaggerUi from 'swagger-ui-express';
```

Reply:
done